test/auth/auth_test.go

```diff
@@ -259,3 +259,29 @@ func getAccessToken(t *testing.T, weaviateUrl, user, pw string) (string, string)
 	require.Nil(t, err)
 	return tokens.AccessToken, tokens.RefreshToken
 }
+
+func TestApiKey(t *testing.T) {
+	t.Run(t.Name(), func(t *testing.T) {
```

Comment:
you don't need the t.Run here. That is only useful if you have several tests inside of one function